services/galley/src/Galley/API.hs

```diff
@@ -573,6 +573,26 @@ sitemap = do
 
     ---
 
+    put "/conversations/:cnv/receipts" (continue updateConversationReceipts) $
```

Comment:
Either it should be a `post`, or the handler should take care of not sending the event when the receipt mode is set to the same value that is already has.